plugins/inventory/cobbler.py

```diff
@@ -107,8 +107,8 @@
         version_added: 7.1.0
       facts_level:
         description:
-          - normal: Gather only system-level variables
-          - as_rendered: Gather all variables as rolled up by Cobbler
+          - normal - Gather only system-level variables
+          - as_rendered - Gather all variables as rolled up by Cobbler
```

Comment:
You can use `:`, you simply have to quote the string correctly to avoid it being a YAML dictionary.

Reply:
Ah okay! Would it be like:

- "normal: Gather only system-level variables"

Reply:
Yes.
```suggestion
          - "as_rendered: Gather all variables as rolled up by Cobbler"
```
or:
```suggestion
          - 'as_rendered: Gather all variables as rolled up by Cobbler'
```
or:
```suggestion
          - >-
            as_rendered: Gather all variables as rolled up by Cobbler
```
or any other way that you can tell YAML that it's a string.

Reply:
Okay, cool - got those changes updated and pushed. I don't think the code is quite done yet, but I just wanted to get all the changed pushed so I don't lose anything.

Orion mentioned the caching part doesn't work and I'm not sure how that works in Ansible